packages/create-cloudflare/templates/common/js/wrangler.toml

```diff
@@ -8,14 +8,19 @@ compatibility_date = "2023-04-21"
 # id = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
 
 # # Durable Object binding - For more information: https://developers.cloudflare.com/workers/runtime-apis/durable-objects
-# [[durable_objects]]
-# binding = "MY_DURABLE_OBJECT"
+# [[durable_objects.bindings]]
+# name = "MY_DURABLE_OBJECT"
 # class_name = "MyDurableObject"
+# # Durable Object migration - For more information: https://developers.cloudflare.com/workers/learning/using-durable-objects#configure-durable-object-classes-with-migrations
+# [[migrations]]
+# tag = "v1"
+# new_classes = ["MyDurableObject"]
 
-# # R2 Bucket binding - For more information: https://developers.cloudflare.com/r2/api/workers/workers-api-reference/#create-a-binding
-# [[r2_buckets]]
+# # Bucket binding - For more information: https://developers.cloudflare.com/workers/runtime-apis/kv#bucket
+# [[buckets]]
```

Comment:
AFAICT it should be `r2_buckets`.

https://developers.cloudflare.com/workers/wrangler/configuration/#r2-buckets
and https://github.com/cloudflare/wrangler2/blob/3b3fadfa16009c6eac2f0aa747db8b06eb80a391/packages/wrangler/src/config/environment.ts#L425